server/svix-server/src/v1/utils.rs

```diff
@@ -256,6 +256,208 @@ fn validation_errors(
         .collect()
 }
 
+/// This is an enum that will wrap every nullable field for a PATCH request. Nonnullable fields can
+/// be represented via an [`Option`] for simplicity. This differs from an [`Option`] in that it
+/// distinguished null values and absent values such that an optional value in a model may be made
+/// None via PATCHing while allowing omitted fields to be skipped when updating.
+///
+/// NOTE: You must tag these fields with `#[serde(default)]` in order for the serialization to work
+/// correctly.
+#[derive(Debug)]
+pub enum NullablePatchField<T> {
+    Absent,
+    None,
+    Some(T),
+}
```

Comment:
To make it much simpler, it should just be:
```suggestion
pub enum NullablePatchField<T> {
    Absent,
    Some(T),
}
```
And then the T can be e.g. String or Option<String>